### PR TITLE
fix queue selection, add optional enviro var passing

### PR DESCRIPTION
**Pull request by gcamilo**

### CODE REVIEW

**Comment by gcamilo** [comment]
@amitmurthy @andreasnoack had to start over, this is the final pull request

**Comment by amitmurthy** [comment]
When working with command objects it is safer to work with command interpolation everywhere and not use string interpolation, i.e, `eVars`, `queue`, `jobname`, etc are themselves command objects.

File: src/qsub.jl [hunk 1]
```diff
@@ -85,5 +91,5 @@ function kill(manager::Union{PBSManager, SGEManager}, id::Int64, config::WorkerC
 
 end
 
-addprocs_pbs(np::Integer, queue="") = addprocs(PBSManager(np, queue))
-addprocs_sge(np::Integer, queue="") = addprocs(SGEManager(np, queue))
+addprocs_pbs(np::Integer; queue::AbstractString="",enviroVars::AbstractString="") = addprocs(PBSManager(np, queue),enviromentVars=enviroVars)
```
Comment (vchuravy) [inline src/qsub.jl]: can we call this `environment` or `enviromentVars` to be consistent?

**Comment by gcamilo** [comment]
Changed to command interpolation and variable names.

**Comment by amitmurthy** [comment]
I have changed the keyword arg to `qsub_env` so that it is more "julian" in style.

**Pull request merged**